src/test/datascience/.vscode/settings.json

```diff
@@ -26,5 +26,6 @@
     // We don't want jupyter to start when testing (DS functionality or anything else).
     "jupyter.disableJupyterAutoStart": true,
     "jupyter.logging.level": "debug",
-    "python.logging.level": "debug"
+    "python.logging.level": "debug",
+    "webview.experimental.useIframes": true
```

Comment:
This was the vscode recommended setting change. They use it in their smoke tests.
https://github.com/microsoft/vscode/issues/125733